src/types/bits.tcc

```diff
@@ -51,6 +51,38 @@ std::array<bool, BitLen> binaryAddition(
     return sum;
 }
 
+// Sum 2 binary strings, discard last carry
+template<size_t BitLen>
+std::array<bool, BitLen> binaryAdditionNoCarry(
```

Comment:
Use `snake_case` for naming

Reply:
Could you also rename the `binaryAddition` too? That way it'll all be consistent.

Reply:
There is already a binaryAddition function which throws an error if there is a carry at the last bit. I was not sure whether we wanted to distinguish the two uses so I created this function.

What would you think of adding a boolean to binaryAddition so that it does not throw an exception when needed ?

Reply:
Yes you can add a flag as arg to the function: https://github.com/clearmatics/zeth/blob/master/src/types/bits.tcc#L28
To determine whether we mod by `2^BitLen` or not. Please set the default value of the argument to be "false" to avoid bad surprises. You'll need to add the flag as part of the conditional to throw the error also.
(and please add a comment above the function explaining the role of the `mod` flag)